src/status_im2/contexts/quo_preview/wallet/account_card.cljs

```diff
@@ -0,0 +1,56 @@
+(ns status-im2.contexts.quo-preview.wallet.account-card
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [quo2.core :as quo]))
+
+(def mock-data
+  [{:id                  1
+    :name                "Alisher account"
+    :balance             "€2,269.12"
+    :percentage-value    "16.9%"
+    :amount              "€570.24"
+    :customization-color :army
+    :type                :default}
+   {:id               2
+    :name             "Ben’s fortune"
+    :balance          "€2,269.12"
+    :percentage-value "16.9%"
+    :amount           "€570.24"
+    :watch-only?      true
+    :type             :watch-only}
+   {:id                  3
+    :type                :add-account
+    :customization-color :blue
+    :handler             #(js/alert "Add account pressed")}])
+
+(defn- separator
+  []
+  [rn/view {:style {:width 40}}])
+
+(defn cool-preview
+  []
+  [rn/view
+   {:style {:margin-vertical 40
+            :margin-left     40
+            :flex            1}}
+   [text/text {:size :heading-1 :weight :semi-bold :style {:margin-bottom 40}} "Account card"]
+   [rn/flat-list
+    {:data                           mock-data
+     :key-extractor                  #(str (:id %))
```

Comment:
just `:d` will do

```
:key-extractor :id
```
since `:id` is already a function as it implements`iFn` https://cljs.github.io/api/cljs.core/IFn 
and I think id is already a string

Reply:
Tks! I will apply it if we decide to also include the flat list to the variants preview